### ISSUE TITLE
For the issue related to PyTorch and CUDA version mismatch encountered during result replication.

### ISSUE THREAD

**Issue by Marceloxo** [xi-jia/Fourier-Net#1]
Thank you for your excellent work. I wanted to ask in what version of CUDA and PyTorch your code was trained and tested. I've been attempting to replicate your results, but I'm encountering issues where the loss doesn't decrease, and the Dice scores on the validation set don't improve. I suspect it might be due to a mismatch in PyTorch versions.

**Comment by xi-jia** [comment]
Thanks.
The models are trained and tested on 2 different machines with details below.

```
> python
Python 3.8.12 (default, Oct 12 2021, 13:49:34) 
[GCC 7.5.0] :: Anaconda, Inc. on linux
Type "help", "copyright", "credits" or "license" for more information.
>>> import torch
>>> torch.__version__
'1.10.0+cu113'
```

```
$ python
Python 3.6.13 |Anaconda, Inc.| (default, Jun  4 2021, 14:25:59)
[GCC 7.5.0] on linux
Type "help", "copyright", "credits" or "license" for more information.
>>> import torch
>>> torch.__version__
'1.10.0+cu102'
```

**Comment by Marceloxo** [comment]
Thank you for your prompt response. For experiments with the 'Fourier-Net' model, do you have a preference for using any of the mentioned environments? Thanks once again!

**Comment by xi-jia** [comment]
Not really, both the two envs produce similar results, as the Pytorch versions are the same.
